### ISSUE TITLE
RS does not show GUI after ubuntu 18.04 update

### ISSUE THREAD

**Issue by POMATu** [RetroShare/RetroShare#1641]
Hi,

So I did a montly update of my Ubuntu 18.04
after reboot I can see password prompt then retroshare loading logo and then nothing happens - no tray icons or windows shown up
no web interface either

When I launch it in terminal - log seems ok

Appimage not working either

Cannot compile from git without that
```
E: Unable to locate package libqtmultimediakit1
E: Unable to locate package qtmobility-dev
```

**Comment by POMATu** [comment]
```
Linux desktop 4.15.0-62-generic #69-Ubuntu SMP Wed Sep 4 20:55:53 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by felisucoibi** [comment]
compile whit this....
https://github.com/felisucoibi/RetroBuild-Ubuntu-18.10-18.04-19.04

**Comment by POMATu** [comment]
@felisucoibi 
Hi Thanks for the tip

However I still cannot compile it with your script
```
make[1]: Entering directory '/home/user/retroshare/retroshare/master/retroshare-gui/src'
make[1]: *** No rule to make target '/home/user/retroshare/retroshare/master/libretroshare/src/lib/libretroshare.a', needed by 'retroshare'.  Stop.
make[1]: Leaving directory '/home/user/retroshare/retroshare/master/retroshare-gui/src'
Makefile:163: recipe for target 'retroshare_gui-install_subtargets' failed
make: *** [retroshare_gui-install_subtargets] Error 2
```
I checked if submodules are checked out - command returns nothing so they are fine if there are any
```
git submodule foreach --recursive git checkout master
```

**Comment by felisucoibi** [comment]
Maybe i missed any module or a new pr made this i ll try tonight.

**Comment by POMATu** [comment]
So I compiled tarball with your script and it works fine but nothing changed

I was able to install retroshare-nogui and at least web interface works now. I wonder if it can be caused by "start minimized" setting that screwed up everything after update.

There is an option to start minimized in retroshare arguments. But there is no option to start not minimized lol. Dont wanna reset all settings and cant edit configs cuz they seem encrypted.

**Comment by probonopd** [comment]
> Appimage not working either

Please elaborate or link to issue, thanks.

**Comment by POMATu** [comment]
Hi, so this issue partly solved.

This was caused by StartMinimized=true
This was my setting before ubuntu update. Now I changed it to StartMinimized=false and I can use retroshare gui from repos. However still no tray icon shown. But at least I can use it normally

So seems like the problem was related to tray icon only. Since appimage and compiled from source versions all used my profile - the result was the same (No window shown)
